src/ngMock/angular-mocks.js

```diff
@@ -1251,20 +1251,21 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
    *   data string and returns true if the data is as expected.
    * @param {(Object|function(Object))=} headers HTTP headers or function that receives http header
    *   object and returns true if the headers match the current definition.
+   * @param {(Array)=} keys Array of keys to assign to regex matches in the respond params for the url.
```

Comment:
I think that this needs a bit more explanation for people coming to this documentation without looking at the code.  Perhaps we could have a single section in the ngMock.$httpBackend description that explains what can now be done with regexes and keys then simply refer each fo these parameter descriptions to that section?